shapely/geometry/base.py

```diff
@@ -414,11 +414,20 @@ def buffer(
         --------
         >>> from shapely.wkt import loads
         >>> g = loads('POINT (0.0 0.0)')
-        >>> g.buffer(1.0).area        # 16-gon approx of a unit radius circle  # doctest: +ELLIPSIS
+
+        16-gon approx of a unit radius circle:
+
+        >>> g.buffer(1.0).area  # doctest: +ELLIPSIS
         3.1365484905459...
-        >>> g.buffer(1.0, 128).area   # 128-gon approximation  # doctest: +ELLIPSIS
-        3.141513801144301...
-        >>> g.buffer(1.0, 3).area     # triangle approximation
+
+        128-gon approximation:
+
+        >>> g.buffer(1.0, 128).area  # doctest: +ELLIPSIS
+        3.141513801144...
+
+        triangle approximation:
+
+        >>> g.buffer(1.0, 3).area  # doctest: +ELLIPSIS
         3.0
```

Comment:
Is this ever not 3.0? Given the simple trigonometry, it should be exact. (A related simple case is a square `g.buffer(1.0, 1)` with `.area` 2.0)

Reply:
Good catch, was likely a copy / paste without thinking it through.